lib/RequestModel.js

```diff
@@ -13,6 +13,13 @@ class RequestModel extends EventEmitter {
 		this.log = false;
 	}
 
+	static addGlobalOptions(options) {
```

Comment:
I did an "add" method, in case we want to expose something else in api, similar to "setProxy".